Initial_script.R

```diff
@@ -15,7 +15,9 @@ data <- mtcars #dataset to play with
 # description: miles per gallon by horse power per number of cylinders:
 
 ggplot(data = mtcars)+
-	geom_point(aes(x = hp, y = mpg), col = "orange")+
+	geom_point(aes(x = hp, y = mpg), col = "purple")+
 	geom_facet(~ cyl)+
 	theme_bw()+
-xlab("some label name")
+	ggtitle()+
+  xlab("some label name")+
+  ylab("another label")
```

Comment:
I like your suggestion but maybe make it more specific.
```suggestion
  ylab("specific label")
```